config/webpack/production.js

```diff
@@ -1,3 +1,12 @@
 const environment = require('./environment')
+const { merge } = require('webpack-merge')
+
+const sassLoader = environment.loaders.get('sass');
+
+sassLoader.use.map(function(loader) {
+  if (loader.loader === "css-loader") {
+    loader.options = merge(loader.options, { sourceMap: false });
```

Comment:
I'd wonder if we can just skip `merge` and...

```suggestion
    loader.options.sourceMap = false;
```

Reply:
I compared the result of the two and it does result in the same:

```
{ sourceMap: false, importLoaders: 2, modules: false }
{ sourceMap: false, importLoaders: 2, modules: false }
```

It seems like we could safely do this and drop the extra package